google/cloud/bigtable/instance_admin.cc

```diff
@@ -321,6 +321,42 @@ btproto::AppProfile InstanceAdmin::UpdateAppProfileImpl(
   return result;
 }
 
+::google::iam::v1::Policy InstanceAdmin::GetIamPolicy(
```

Comment:
Since we went through the trouble of creating IAM wrappers for bindings: should we return a wrapper here instead of the raw proto?  Can you create a bug and put a TODO() to think about that?